src/validator.coffee

```diff
@@ -218,11 +222,18 @@ exports.validate = (inputAddr, addressType=defaultMatchType, cb) ->
     qs.key = options.key
     protocol = 'https'
 
-  opts =
-      json: true,
-      url: "#{protocol}://maps.googleapis.com/maps/api/geocode/json"
-      method: 'GET'
-      qs: qs
+  endpoint = "#{protocol}://maps.googleapis.com/maps/api/geocode/json?#{querystring.stringify(qs)}";
+
+  opts = url.parse(endpoint);
+  opts.url = endpoint;
```

Comment:
perhaps what you actually wanted to do here was `opt.url = url.parse(endpoint).href` ?

https://nodejs.org/docs/latest/api/url.html#url_urlobject_href

Reply:
Ah, yes - thank you. I realized right after I made the PR that it was probably not necessary to do this, as I was going off of the [example](https://www.npmjs.com/package/https-proxy-agent#https-module-example) provided in `https-proxy-agent`'s Readme which uses `http.get` directly. And in this case, `request` would handle this transition. I will revert it to more closely resemble your initial setup of `opts`... as well as update the readme as you've requested.